etc/kayobe/networks.yml

```diff
@@ -33,7 +33,9 @@ internal_net_name: internal
 # Deprecated name: external_net_name
 # If external_net_name is defined, external_net_names will default to a list
 # containing one item, external_net_name.
-external_net_names: []
+external_net_names:
+  # This causes the openvswitch to be created
+  - oc_provision
```

Comment:
If we used the public network here instead, would that have the same effect?

Reply:
Currently, any network that is added here will get an IP on the compute nodes. We don't want that for the public network, so we use a network that it has already.